app/auth/forms.py

```diff
@@ -99,6 +102,75 @@ def validate(self):
             ))
 
 
+class ManageAgencyUserAccountForm(StripFieldsForm):
+    """
+    This form manages the OpenRecords specific account fields for a user account:
+        Title: The job title for the user (e.g. Reporter); This is optional
+        Organization: The company for the user (e.g. New York Times); This is optional
+        Notification Email: The email to send OpenRecords notifications to
+        Phone Number: The user's phone number; This is optional
+        Fax Number: The user's fax number: This is optional
+        Mailing Address: The user's mailing address; Optional;
+            Format: Address One, Address Two, City, State, Zip (5 Digits)
+    """
+    default_agency = SelectField('Primary Agency', validators=[DataRequired()])
+    title = StringField('Title', validators=[Length(max=64), Optional()])
+    organization = StringField('Organization', validators=[Length(max=254), Optional()])
+    notification_email = StringField('Notification Email', validators=[Email(), Length(max=254), Optional()])
+    phone_number = StringField('Phone', validators=[Length(min=10, max=25), Optional()])
+    fax_number = StringField('Fax', validators=[Length(min=10, max=25), Optional()])
+    address_one = StringField('Line 1', validators=[Optional()])
+    address_two = StringField('Line 2', validators=[Optional()])
+    city = StringField('City', validators=[Optional()])
+    state = SelectField('State', choices=STATES, default='NY', validators=[Optional()])
+    zipcode = StringField('Zip Code (5 Digits)', validators=[Length(min=5, max=5), Optional()])
+
+    submit = SubmitField('Update OpenRecords Account')
+
+    def __init__(self, user=None):
+        """
+        :type user: app.models.Users
+        """
+        super(ManageAgencyUserAccountForm, self).__init__()
+        self.user = user
+        self.default_agency.choices = [
+            (a.ein, a.name)
+            for a in user.agencies.all()]
+        for agency in self.default_agency.choices:
```

Comment:
Since this code is re-used multiple times, make this a function in the Users models that querys for all the agencies in the database or just the agencies that the user is a member of.